tgui/packages/tgui/hotkeys.ts

```diff
@@ -135,9 +139,9 @@ export const releaseHeldKeys = () => {
 
 export const setupHotKeys = () => {
   // Read macros
-  Byond.winget('default.*').then(data => {
+  Byond.winget('default.*').then((data: any) => {
     // Group each macro by ref
-    const groupedByRef = {};
+    const groupedByRef: any = {};
```

Comment:
Same here--we control this so I'm not happy about sticking `any` on it. Looks like `Record<string, Record<string, ByondHotkey>>`.

Reply:
Happy now? I really didn't want to do this, I tried my best, this code is hard to strongly type, and there is no guarantee data is a `Record<string, string>`, more like `Record<string, unknown>`. See where this is going?